### PR TITLE
react-react1-week1

### PR DESCRIPTION
**Pull request by alina-kho**

### CODE REVIEW

File: react/react1/week1/todo-list/src/components/toDoList.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import ToDoItem from "./toDoItem";
+
+export default function ToDoList() {
+  return (
+    <div className="toDoList">
+      <ul>
+        {todos.map((todo) => {
+          return (
+            <ToDoItem
+              key={todo.id}
```
Comment (dsamourkasidis) [inline react/react1/week1/todo-list/src/components/toDoList.js]: You are using an id property which, if I look at your todos.js file doesn't seem to exist!
So either add it there, eg
```
{ id: 1, description: "Get out of bed", deadline: "Wed Sep 13 2017" },
```

or generate a random id to use as key

File: react/react1/week1/todo-list/src/components/toDoList.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import todos from "../data/todos";
+import ToDoItem from "./toDoItem";
+
+export default function ToDoList() {
```
Comment (dsamourkasidis) [inline react/react1/week1/todo-list/src/components/toDoList.js:4]: Minor detail ->
Your component name is ToDoList but the file name is toDoList.
Try to keep it consistent and rename the file as well to ToDoList.js
Same with ToDoItem.js

**Review by dsamourkasidis — COMMENTED** [review]
Great work 👍 
Nice job in breaking down the components to ToDoList and ToDoItem and passing down props.
I just left two small comments I believe you will address quite easily!

File: react/react1/week1/todo-list/src/components/toDoList.js (hunk 1 above)
Comment (alina-kho) [reply react/react1/week1/todo-list/src/components/toDoList.js]: Hi Dimitris :) Thanks a lot for the feedback, I have applied the improvements you mentioned 🤓

**Pull request merged**